Examples/BookingSystem.AspNetCore/Properties/launchSettings.json

```diff
@@ -10,7 +10,7 @@
       "launchUrl": "https://localhost:5001/openactive",
       "applicationUrl": "https://localhost:5001",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_ENVIRONMENT": "no-auth",
```

Comment:
Suspect this shouldn't have been left in?

Reply:
Have removed that in the next commit